tuf/api/serialization/json.py

```diff
@@ -0,0 +1,55 @@
+"""TUF role metadata JSON serialization and deserialization.
+
+This module provides concrete implementations to serialize and deserialize TUF
+role metadata to and from the JSON wireline format for transportation, and
+to serialize the 'signed' part of TUF role metadata to the OLPC Canonical JSON
+format for signature generation and verification.
+
+"""
+import json
+
+from securesystemslib.formats import encode_canonical
+
+from tuf.api.metadata import Metadata, Signed
+from tuf.api.serialization import (MetadataSerializer,
+                                   MetadataDeserializer,
+                                   SignedSerializer)
```

Comment:
I know. I guess I just kept using the convention that we already used in this file, which we started before fully embracing our new style guide... and because PEP 8 says it's okay.

Reply:
I agree with Joshua on this one, we don't have a linter that would automatically catch those mistakes,
but it's important we correct each other.
Also, it seems logical to forbid importing of functions and classes.
This way we prevent future problems with naming and namespaces.